wrapper/src/test/java/integration/refactored/container/tests/HikariTests.java

```diff
@@ -111,4 +142,185 @@ public void testOpenConnectionWithDataSourceClassName() throws SQLException {
     assertTrue(conn.isValid(10));
     conn.close();
   }
+
+  /**
+   * After getting successful connections from the pool, the cluster becomes unavailable.
+   */
+  @TestTemplate
+  @EnableOnTestFeature(TestEnvironmentFeatures.FAILOVER_SUPPORTED)
+  public void test_1_1_hikariCP_lost_connection() throws SQLException {
```

Comment:
Probably we need a better name for this test. Reference to 1_1 something doesn't make sense any more.